src/amo/actions/reviews.js

```diff
@@ -311,7 +341,11 @@ export function fetchUserReviews({
   errorHandlerId,
   userId,
   page = '1',
-}: FetchUserReviewsParams): FetchUserReviewsAction {
+}: {|
```

Comment:
> Why did you decide to use an explicit type for FetchLatestUserReviewParams 

because it's shared by the action payload

> ...but do away with the explicit type here and just create an inline type definition?

because this one is not shared with anything. I find inline object types preferable; you can read the code top to bottom without having to look up a reference to the type.